<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayOptions</class>
 <widget class="QWidget" name="DisplayOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Display options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>settingLabel</zorder>
   <zorder>back</zorder>
  </widget>
  <widget class="QWidget" name="checkboxes" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>300</width>
     <height>180</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="enableRotation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Room rotation enabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableNightmode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nightmode enabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableSounds">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sounds enabled</string>
    </property>
   </widget>
   <widget class="QPushButton" name="questionnaireOptions">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Questionnaire options</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
